src/App.js

```diff
@@ -1,26 +1,45 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react'
 import './App.css';
+import Home from './components/home/Home';
+import { Route, Switch } from 'react-router-dom'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import RandomBeer from './components/randombeer/RandomBeer';
+import NewBeer from './components/newbeer/NewBeer';
+import Header from './components/header/Header';
+import AllBeers from './components/allbeers/AllBeers';
+import BeerDetail from './components/beerdetail/BeerDetail';
+
+
+export default class App extends Component {
+
+  state = {
+    beers: ''
+  }
+
+  updateStateAppJs = (beersFromDB) => {
```

Comment:
To keep the state of the beers also in App.js is an extra step which can create some confusion. It's enough to have it just in the AllBeers component